catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/controller/AbstractEventController.java

```diff
@@ -57,7 +59,12 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractEventController.class);
 
-    private final ExecutorService executorService = Executors.newCachedThreadPool();
+    private final ExecutorService executorService = new ThreadPoolExecutor(0,
```

Comment:
❓ Is there a reason you changed this to manual construction of your threadpool instead of just calling Executors.newCachedThreadPool(StandardThreadFactoryBuilder.newThreadFactory("eventProcessorThread"))

Reply:
`newCachedThreadPool` will work here.